.github/workflows/ci.yaml

```diff
@@ -1,19 +1,18 @@
+---
 name: CI
-
 on:
   push:
   pull_request:
   workflow_dispatch:
   schedule:
-  - cron: '0 0 * * *'
-
+  - cron: 0 0 * * *
 jobs:
   ci:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         ruby-version:
-        - '2.7'
+        - '3.0'
```

Comment:
Why did we drop 2.7?  I thought we always wanted to keep one version back in case we have to revert?

Reply:
Oh I see now - the other comment was collapsed.

Reply:
We didn't drop 2.7.

We run the js specs with the latest, then also the `spec` test_suite with 2.7 https://github.com/ManageIQ/manageiq-ui-classic/pull/8456/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR26

Reply:
This is the same behavior as it was previously, but with 3.0, 2.7 instead of 2.7, 2.6